corehq/apps/hqwebapp/models.py

```diff
@@ -1389,12 +1389,15 @@ def sidebar_items(self):
         admin_operations = []
 
         if self.couch_user and self.couch_user.is_staff:
```

Comment:
Anyone know what the use is of the `is_staff` permission. It seems superfluous if the user is already super_user